src/applications/vaos/reducers/newAppointment.js

```diff
@@ -474,31 +476,25 @@ export default function formReducer(state = initialState, action) {
           [action.facilityId]: action.facilityDetails,
         },
       };
-    case FORM_SCHEDULE_APPOINTMENT_PAGE_OPENED: {
+    case FORM_FETCH_AVAILABLE_APPOINTMENTS: {
       return {
         ...state,
-        loadingAppointmentSlots: true,
-        availableSlots: [],
-        appointmentLength: null,
+        appointmentSlotsStatus: FETCH_STATUS.loading,
```

Comment:
Can we add `appointmentSlotsStatus` to the `initialState` object, with a `notStarted` status?